src/externs.ts

```diff
@@ -616,7 +616,7 @@ export function generateExterns(
    * cases.
    */
   function debugLocationStr(node: ts.Node, namespace: ReadonlyArray<string>) {
-    return namespace.join('.') || path.basename(node.getSourceFile().fileName);
+    return namespace.join('.') || node.getSourceFile().fileName.replace(/.*[/\\]/, '');
```

Comment:
add a comment because regexps are hard to read.

Reply:
Done.